usaspending_api/disaster/v2/views/overview.py

```diff
@@ -147,10 +147,7 @@ def additional_totals() -> Optional[dict]:
         ]
         outlay_values = ["gross_outlay_amount_by_tas_cpe", "anticipated_prior_year_obligation_recoveries"]
         total_budget_authority_values = [
-            "total_budgetary_resources_cpe",
-            "budget_authority_unobligated_balance_brought_forward_cpe",
             "deobligations_or_recoveries_or_refunds_from_prior_year_cpe",
```

Comment:
This should be removed as well since it is mentioned in obligation_values and tba_values.